src/PlayBack/playbackUtil.tsx

```diff
@@ -0,0 +1,19 @@
+const THUMB_HALF_WIDTH = 7;
```

Comment:
Double check the `any` types in the file. I think they can really be `number` most of the time.